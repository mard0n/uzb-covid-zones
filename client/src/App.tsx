import React, { Suspense, useEffect, useState } from "react";
import {
  Route,
  BrowserRouter as Router,
  Switch,
  Redirect,
} from "react-router-dom";
import StateContextProvider, { initialState } from "./state/StateContext";
import auth from "./utils/auth";
import ProtectedRoute from "./routes/ProtectedRoute";
import {
  ThemeProvider,
  createMuiTheme,
  useMediaQuery,
  Theme,
} from "@material-ui/core";
import RubikRegularWoff2 from "./assets/fonts/Rubik-Regular.woff2";
import RubikRegularWoff from "./assets/fonts/Rubik-Regular.woff";
import RubikRegularTtf from "./assets/fonts/Rubik-Regular.ttf";
import RubikMediumWoff2 from "./assets/fonts/Rubik-Medium.woff2";
import RubikMediumWoff from "./assets/fonts/Rubik-Medium.woff";
import RubikMediumTtf from "./assets/fonts/Rubik-Medium.ttf";

const Login = React.lazy(() => import("./screens/Login"));
const Admin = React.lazy(() => import("./screens/Admin"));
const User = React.lazy(() => import("./screens/User"));

/* rubik-regular - latin_cyrillic */
const raleway = {
  fontFamily: "Rubik",
  fontStyle: "normal",
  fontWeight: 400,
  src: `
    local('Raleway'),
    local('Raleway-Regular'),
    url(${RubikRegularWoff2}) format('woff2'),
    url(${RubikRegularWoff}) format('woff'),
    url(${RubikRegularTtf}) format('truetype')
  `,
  // unicodeRange:
  //   'U+0000-00FF, U+0131, U+0152-0153, U+02BB-02BC, U+02C6, U+02DA, U+02DC, U+2000-206F, U+2074, U+20AC, U+2122, U+2191, U+2193, U+2212, U+2215, U+FEFF',
};
const RubikRegular = {
  fontFamily: "Rubik",
  fontStyle: "normal",
  fontWeight: 400,
  src: `
    local('Rubik'),
    local('Rubik-Regular'),
    url(${RubikRegularWoff2}) format('woff2'),
    url(${RubikRegularWoff}) format('woff'), 
    url(${RubikRegularTtf}) format('truetype')`,
};
/* rubik-500 - latin_cyrillic */
const RubikMedium = {
  fontFamily: "Rubik",
  fontStyle: "normal",
  fontWeight: 500,
  src: `
    local('Rubik Medium'),
    local('Rubik-Medium'),
    url(${RubikMediumWoff2}) format('woff2'),
    url(${RubikMediumWoff}) format('woff'),
    url(${RubikMediumTtf}) format('truetype')`,
};

const theme = createMuiTheme({
  typography: {
    fontFamily: "Rubik",
    h1: {
      fontSize: 26,
      fontWeight: 500,
    },
    h2: {
      fontSize: 24,
      fontWeight: 500,
    },
    h3: {
      fontSize: 22,
      fontWeight: 500,
    },
    h4: {
      fontSize: 20,
      fontWeight: 500,
    },
    h5: {
      fontSize: 18,
      fontWeight: 500,
    },
    subtitle1: {
      fontSize: 18,
      fontWeight: 500,
    },
    body1: {
      fontSize: 16,
      fontWeight: 500,
    },
    body2: {
      fontSize: 16,
      fontWeight: 400,
    },
    caption: {
      fontSize: 14,
      fontWeight: 400,
    },
    overline: {
      fontSize: 12,
      fontWeight: 500,
    },
  },
  overrides: {
    MuiCssBaseline: {
      "@global": {
        "@font-face": [RubikRegular, RubikMedium],
      },
    },
  },
  palette: {
    primary: {
      main: "#4863F4",
    },
    secondary: {
      main: "#F0F3FE",
      contrastText: "#2E409E",
    },
    text: {
      primary: "#242B43",
      secondary: "#777FA9",
    },
    grey: {
      "500": "#969BAC",
    },
  },
});

// theme.typography.h1 = {
//   fontFamily: 'Rubik',
//   fontWeight: 500,
//   letterSpacing: "-0.01562em",
//   lineHeight: 1.167,
//   fontSize: 24,
//   [theme.breakpoints.up("md")]: {
//     fontSize: 26,
//   },
// };
// theme.typography.body2 = {
//   // fontFamily: 'Rubik',
//   fontFamily: '"Rubik", "Arial", "Helvetica", sans-serif',
//   fontWeight: 400,
//   letterSpacing: "0.01071em",
//   lineHeight: 1.43,
//   fontSize: 14,
//   [theme.breakpoints.up("md")]: {
//     fontSize: 16,
//   },
// };

function App() {
  console.log("theme ", theme);
  return (
    <StateContextProvider initialState={initialState}>
      <ThemeProvider theme={theme}>
        <Router>
          <Suspense fallback={<div>Loading... </div>}>
            <Switch>
              <Route path={"/app"} component={User} />
              {/* <ProtectedRoute
              path={"/admin"}
              component={Admin}
            /> */}
              <Route path={"/admin"} component={Admin} />
              <Route path={"/login"} component={Login} />
              <Route path="*">Not found</Route>
            </Switch>
          </Suspense>
        </Router>
      </ThemeProvider>
    </StateContextProvider>
  );
}

export default App;
