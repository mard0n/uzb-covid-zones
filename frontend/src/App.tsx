import { Layout, EmbedLayout } from "./layouts";
import { createBrowserRouter, RouterProvider } from "react-router-dom";
import ErrorPage from "./error-page";
import { ZoneResType } from "./types/zone";

async function rootloader() {
  const urlSearchParams = new URLSearchParams(window.location.search);
  const params = urlSearchParams.getAll("zone");

  if (params.length) {
    const res: ZoneResType = await fetch(
      `api/zones?${params.map((p) => "zone=" + p).join("&")}`
    ).then((res) => res.json());
    if (res?.zones) {
      return res.zones;
    }
  } else {
    const res: ZoneResType = await fetch(`api/zones`).then((res) => res.json());
    if (res?.zones) {
      return res.zones;
    }
  }
}

const router = createBrowserRouter([
  {
    path: "/",
    element: <Layout />,
    errorElement: <ErrorPage />,
    loader: rootloader,
  },
  {
    path: "/embed",
    element: <EmbedLayout />,
    errorElement: <ErrorPage />,
    loader: rootloader,
  },
]);

function App() {
  return <RouterProvider router={router} />;
}

export default App;

// TODO: Add select all children Button in embed screen
// TODO: Minify Zones
// TODO: Test on interactivity, drag, pan on different devices