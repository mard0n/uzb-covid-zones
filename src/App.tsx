import React, { FunctionComponent } from "react";
import Routes from "./router";
import { AppProps } from "./types";
import {
  Footer,
  Main,
  Header,
  LinearProgressBar,
  Select,
  Box,
  Caption
} from "@mashreq-digital/ui";

import { connect } from "react-redux";
import { stepsID } from "./reducers/createAcountReducer";
import { getMashreqLogo } from "@mashreq-digital/webassets";
const App: FunctionComponent<AppProps & { activeStepTitle: string }> = (
  props: any
): JSX.Element => {
  const { activeStep, activeStepTitle } = props;

  console.log("stepsID", stepsID.length);
  console.log("activeStep", activeStep);

  const MashreqLogo = getMashreqLogo();

  return (
    <Main
      header={
        <div>
          <Header
            left={
              <Box display="flex" alignItems="center">
              <MashreqLogo width="40px" height="25px" />
                {props.match.url === "/login" ? null : (
                  <Box ml={2.5}>
                    <Caption>{activeStepTitle}</Caption>
                  </Box>
                )}
              </Box>
            }
            right={
              <Box>
                {props.match.url === "/login" ? (
                  <Select native onChange={() => {}}>
                    <option value="English">English</option>
                    <option value="Arabic">Arabic</option>
                  </Select>
                ) : (
                  <Caption> Need Help ? </Caption>
                )}
              </Box>
            }
          />

          {props.match.url === "/login" ? null : (
            <LinearProgressBar
              activeStep={activeStep}
              variant="determinate"
              totalStep={stepsID.length - 1}
            />
          )}
        </div>
      }
      main={<Routes />}
      footer={
        <Footer>
          <Box>
            <Box style={{float:"left"}}>  
            <Caption> Â© 2019 Mashreq </Caption>
            </Box>
            <Box style={{float:"right"}}>
              <Caption>
                Requirements | Security | Privacy policy | Service Terms{" "}
              </Caption>
            </Box>
          </Box>
        </Footer>
      }
    />
  );
};

const mapStateToProps = (state: any) => ({
  activeStepTitle: state.createAccount.activeStepTitle,
  activeStep: state.createAccount.activeStep
});

App.defaultProps = {
  activeStepTitle: "default"
};

export default connect(mapStateToProps)(App);
