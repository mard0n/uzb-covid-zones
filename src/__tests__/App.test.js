import React from 'react';
import { render, cleanup } from "@testing-library/react";
import App from '../App';

afterEach(cleanup);

it("matches snapshot", () => {
  const { asFragment } = render( <App/> );
  expect(asFragment()).toMatchSnapshot();
});