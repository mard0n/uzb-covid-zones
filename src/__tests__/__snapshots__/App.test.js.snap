// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="MuiBox-root MuiBox-root-4"
  >
    <div
      class="MuiGrid-root makeStyles-root-5 makeStyles-root-7 MuiGrid-container"
    >
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-12 MuiGrid-grid-md-12 MuiGrid-grid-lg-12 MuiGrid-grid-xl-12"
      >
        <div
          class="MuiBox-root MuiBox-root-114 makeStyles-root-111 "
        >
          <div
            class="Header makeStyles-grow-118"
          >
            <header
              class="MuiPaper-root MuiAppBar-root MuiAppBar-positionFixed MuiAppBar-colorPrimary makeStyles-appBar-115 mui-fixed MuiPaper-elevation4"
            >
              <div
                class="MuiToolbar-root MuiToolbar-regular makeStyles-toolbar-116 MuiToolbar-gutters"
              >
                <div
                  class="makeStyles-flexGrow-117"
                >
                  <div
                    class="MuiBox-root MuiBox-root-162"
                  >
                    <svg
                      height="40"
                      viewBox="0 0 91 44"
                      width="80"
                    >
                      <defs>
                        <lineargradient
                          id="Logos_Mashreq_symbol_svg__a"
                          x1="16.934%"
                          x2="100%"
                          y1="50%"
                          y2="50%"
                        >
                          <stop
                            offset="0%"
                            stop-color="#E85236"
                          />
                          <stop
                            offset="48.2%"
                            stop-color="#F49E30"
                          />
                          <stop
                            offset="100%"
                            stop-color="#EFE300"
                          />
                        </lineargradient>
                        <lineargradient
                          id="Logos_Mashreq_symbol_svg__b"
                          x1="-8.33%"
                          x2="115.535%"
                          y1="32.642%"
                          y2="68.696%"
                        >
                          <stop
                            offset="0%"
                            stop-color="#E85236"
                          />
                          <stop
                            offset="48.2%"
                            stop-color="#F49E30"
                          />
                          <stop
                            offset="100%"
                            stop-color="#EFE300"
                          />
                        </lineargradient>
                        <lineargradient
                          id="Logos_Mashreq_symbol_svg__c"
                          x1="28.643%"
                          x2="67.441%"
                          y1="-28.7%"
                          y2="113.091%"
                        >
                          <stop
                            offset="0%"
                            stop-color="#E85236"
                          />
                          <stop
                            offset="48.2%"
                            stop-color="#F49E30"
                          />
                          <stop
                            offset="100%"
                            stop-color="#EFE300"
                          />
                        </lineargradient>
                        <lineargradient
                          id="Logos_Mashreq_symbol_svg__d"
                          x1="51.719%"
                          x2="49.364%"
                          y1="-15.755%"
                          y2="120.799%"
                        >
                          <stop
                            offset="0%"
                            stop-color="#E85236"
                          />
                          <stop
                            offset="48.2%"
                            stop-color="#F49E30"
                          />
                          <stop
                            offset="100%"
                            stop-color="#EFE300"
                          />
                        </lineargradient>
                        <lineargradient
                          id="Logos_Mashreq_symbol_svg__e"
                          x1="89.178%"
                          x2="11.363%"
                          y1="-29.754%"
                          y2="122.437%"
                        >
                          <stop
                            offset="0%"
                            stop-color="#E85236"
                          />
                          <stop
                            offset="48.2%"
                            stop-color="#F49E30"
                          />
                          <stop
                            offset="100%"
                            stop-color="#EFE300"
                          />
                        </lineargradient>
                        <lineargradient
                          id="Logos_Mashreq_symbol_svg__f"
                          x1="118.022%"
                          x2="-15.179%"
                          y1="33.611%"
                          y2="65.78%"
                        >
                          <stop
                            offset="0%"
                            stop-color="#E85236"
                          />
                          <stop
                            offset="48.2%"
                            stop-color="#F49E30"
                          />
                          <stop
                            offset="100%"
                            stop-color="#EFE300"
                          />
                        </lineargradient>
                      </defs>
                      <g
                        fill="none"
                        fill-rule="evenodd"
                      >
                        <path
                          d="M0 37.178c9.945 4.374 19.617 6.682 28.99 6.701h.235c9.073-.018 17.868-2.186 26.355-6.701-9.31-4.238-18.663-6.463-28.072-6.463C18.39 30.716 9.22 32.806 0 37.178"
                          fill="url(#Logos_Mashreq_symbol_svg__a)"
                        />
                        <path
                          d="M58.506 30.676c-8.69-12.852-21.115-20.068-38.03-20.438 9.128 13.953 21.71 21.079 38.03 20.438"
                        />
                        <path
                          d="M58.506 30.676c-8.69-12.852-21.115-20.068-38.03-20.438 9.128 13.953 21.71 21.079 38.03 20.438"
                          fill="url(#Logos_Mashreq_symbol_svg__b)"
                        />
                        <path
                          d="M46.97 0c-1.142 12.109 3.626 21.525 14.287 27.79C62.4 16.633 57.94 7.246 46.97 0z"
                          fill="url(#Logos_Mashreq_symbol_svg__c)"
                        />
                        <path
                          d="M66.944 26.327c5.687-7.096 6.387-14.891 1.3-23.402-6.678 7.454-7.098 15.36-1.3 23.402"
                          fill="url(#Logos_Mashreq_symbol_svg__d)"
                        />
                        <path
                          d="M73.156 27.79c7.626-2.021 11.559-7.348 11.677-16.089-8.071 2.025-12.05 7.434-11.677 16.09"
                          fill="url(#Logos_Mashreq_symbol_svg__e)"
                        />
                        <path
                          d="M76.057 30.393c6.985 1.168 12.166-.792 14.627-6.676a20.055 20.055 0 00-3.435-.315c-5.097 0-8.895 2.311-11.192 6.991"
                          fill="url(#Logos_Mashreq_symbol_svg__f)"
                        />
                      </g>
                    </svg>
                  </div>
                </div>
                <div
                  class="makeStyles-flexGrow-117"
                />
                <div>
                  <div
                    class="MuiBox-root MuiBox-root-163"
                  >
                    <div
                      class="MuiInputBase-root MuiInput-root MuiInput-underline"
                    >
                      <select
                        class="MuiSelect-root MuiSelect-select MuiInputBase-input MuiInput-input"
                      >
                        <option
                          value="er"
                        >
                          English
                        </option>
                        <option
                          value="ar"
                        >
                          عربى
                        </option>
                      </select>
                      <svg
                        aria-hidden="true"
                        class="MuiSvgIcon-root MuiSelect-icon"
                        focusable="false"
                        role="presentation"
                        viewBox="0 0 24 24"
                      >
                        <path
                          d="M7 10l5 5 5-5z"
                        />
                      </svg>
                    </div>
                  </div>
                </div>
              </div>
            </header>
          </div>
        </div>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-12 MuiGrid-grid-md-12 MuiGrid-grid-lg-12 MuiGrid-grid-xl-12"
      >
        <div
          class="MuiBox-root MuiBox-root-215"
        >
          <div
            class="MuiBox-root MuiBox-root-216"
          >
            <div
              class="MuiGrid-root makeStyles-root-219 makeStyles-root-221 MuiGrid-container"
            >
              <div
                class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-12 MuiGrid-grid-md-7 MuiGrid-grid-lg-7 MuiGrid-grid-xl-7"
              >
                <div
                  class="MuiBox-root MuiBox-root-223 makeStyles-contentWrapper-220 makeStyles-contentWrapper-222"
                >
                  <div
                    class="MuiBox-root MuiBox-root-224"
                  >
                    <div
                      class="MuiBox-root MuiBox-root-227 makeStyles-root-225 makeStyles-root-226"
                    >
                      <div
                        class="MuiBox-root MuiBox-root-228"
                      >
                        <div
                          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-10 MuiGrid-grid-sm-10 MuiGrid-grid-md-10 MuiGrid-grid-lg-10 MuiGrid-grid-xl-10"
                        >
                          <div
                            class="MuiBox-root MuiBox-root-236"
                          >
                            <h1
                              class="MuiTypography-root MuiTypography-h1"
                            >
                              Welcome to Mashreq Online Banking
                            </h1>
                            <div
                              class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-7 MuiGrid-grid-sm-7 MuiGrid-grid-md-7 MuiGrid-grid-lg-7 MuiGrid-grid-xl-7"
                            >
                              <div
                                class="MuiFormGroup-root"
                              >
                                <div
                                  class="MuiFormControl-root makeStyles-inputBox-218"
                                >
                                  <div
                                    aria-describedby="Username"
                                    class="MuiFormControl-root MuiTextField-root makeStyles-root-273 makeStyles-root-278"
                                  >
                                    <label
                                      class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-shrink MuiInputLabel-filled Mui-focused Mui-focused"
                                      data-shrink="true"
                                      for="username"
                                      id="username-label"
                                    >
                                      Username
                                    </label>
                                    <div
                                      class="MuiInputBase-root MuiFilledInput-root Mui-focused Mui-focused MuiInputBase-formControl MuiInputBase-adornedEnd MuiFilledInput-adornedEnd"
                                    >
                                      <input
                                        aria-invalid="false"
                                        aria-label="Username"
                                        class="MuiInputBase-input MuiFilledInput-input MuiInputBase-inputAdornedEnd MuiFilledInput-inputAdornedEnd"
                                        id="username"
                                        maxlength="80"
                                        type="text"
                                        value=""
                                      />
                                    </div>
                                  </div>
                                </div>
                                <div
                                  class="MuiFormControl-root makeStyles-inputBox-218"
                                >
                                  <div
                                    class="MuiFormControl-root MuiTextField-root makeStyles-root-273 makeStyles-root-316"
                                  >
                                    <label
                                      class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-filled"
                                      data-shrink="false"
                                      for="password"
                                      id="password-label"
                                    >
                                      Password
                                    </label>
                                    <div
                                      class="MuiInputBase-root MuiFilledInput-root MuiInputBase-formControl MuiInputBase-adornedEnd MuiFilledInput-adornedEnd"
                                    >
                                      <input
                                        aria-invalid="false"
                                        aria-label="Password"
                                        class="MuiInputBase-input MuiFilledInput-input MuiInputBase-inputAdornedEnd MuiFilledInput-inputAdornedEnd"
                                        id="password"
                                        maxlength="80"
                                        type="password"
                                        value=""
                                      />
                                      <div
                                        class="MuiInputAdornment-root MuiInputAdornment-filled MuiInputAdornment-positionEnd"
                                      >
                                        <button
                                          aria-label="Password"
                                          class="MuiButtonBase-root MuiIconButton-root"
                                          tabindex="0"
                                          type="button"
                                        >
                                          <span
                                            class="MuiIconButton-label"
                                          >
                                            <svg
                                              height="1em"
                                              viewBox="0 0 24 24"
                                              width="1em"
                                            >
                                              <path
                                                d="M17.499 1c3.584 0 6.5 2.916 6.5 6.5a6.481 6.481 0 01-1.688 4.366c.475.453.953.944 1.425 1.459.35.381.35.969 0 1.353-1.9 2.072-3.837 3.653-5.759 4.703C16.005 20.453 13.993 21 11.999 21c-1.994 0-4.006-.547-5.978-1.622-1.922-1.047-3.859-2.631-5.759-4.703a1 1 0 010-1.35c1.816-1.981 3.669-3.516 5.503-4.559 1.731-.987 3.497-1.563 5.25-1.722A6.508 6.508 0 0117.499 1zM8.141 9.822c-1.906.806-3.834 2.203-5.763 4.178C5.615 17.319 8.85 19 12 19c3.15 0 6.384-1.681 9.625-5a26.446 26.446 0 00-.884-.866 6.473 6.473 0 01-3.835.838 4.989 4.989 0 01-1.525 2.712A4.995 4.995 0 0112 18c-2.756 0-5-2.244-5-5a4.98 4.98 0 011.141-3.178zm3.053-.741A3.976 3.976 0 008 13c0 2.206 1.794 4 4 4a3.976 3.976 0 003.919-3.194 6.526 6.526 0 01-4.725-4.725zM17.5 2A5.508 5.508 0 0012 7.5c0 3.031 2.469 5.5 5.5 5.5S23 10.531 23 7.5 20.531 2 17.5 2zm2 2.794l.706.706-2 2 2 2-.706.706-2-2-2 2-.706-.706 2-2-2-2 .706-.706 2 2 2-2z"
                                                fill="currentColor"
                                                fill-rule="evenodd"
                                              />
                                            </svg>
                                          </span>
                                          <span
                                            class="MuiTouchRipple-root"
                                          />
                                        </button>
                                      </div>
                                    </div>
                                  </div>
                                </div>
                                <div
                                  class="MuiBox-root MuiBox-root-336"
                                >
                                  <label
                                    class="MuiFormControlLabel-root makeStyles-inputBox-218"
                                  >
                                    <span
                                      class="MuiSwitch-root makeStyles-root-343 makeStyles-root-350 MuiSwitch-sizeSmall makeStyles-sizeSmall-345 makeStyles-sizeSmall-351"
                                    >
                                      <span
                                        aria-disabled="false"
                                        class="MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-366 MuiSwitch-switchBase makeStyles-switchBase-344"
                                        variant="ios"
                                      >
                                        <span
                                          class="MuiIconButton-label"
                                        >
                                          <input
                                            class="PrivateSwitchBase-input-369 MuiSwitch-input"
                                            type="checkbox"
                                            value="showKeyboard"
                                          />
                                          <span
                                            class="MuiSwitch-thumb makeStyles-thumb-346 makeStyles-thumb-352"
                                          />
                                        </span>
                                      </span>
                                      <span
                                        class="MuiSwitch-track makeStyles-track-347 makeStyles-track-353"
                                      />
                                    </span>
                                    <span
                                      class="MuiTypography-root MuiFormControlLabel-label MuiTypography-body1"
                                    >
                                      Virtual Keyboard
                                    </span>
                                  </label>
                                </div>
                                <div
                                  class="MuiBox-root MuiBox-root-370"
                                >
                                  <h4
                                    class="MuiTypography-root MuiTypography-h4"
                                  >
                                     Ensure You are Always Protected 
                                  </h4>
                                  <span
                                    class="MuiTypography-root MuiTypography-caption"
                                  >
                                     Check out these security tips to protect yourself from being a victim of online threats.
                                  </span>
                                </div>
                              </div>
                            </div>
                          </div>
                        </div>
                      </div>
                      <div
                        class="MuiBox-root MuiBox-root-371"
                      >
                        <div
                          class="MuiBox-root MuiBox-root-372"
                        >
                          <button
                            class="MuiButtonBase-root MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary"
                            tabindex="0"
                            type="button"
                          >
                            <span
                              class="MuiButton-label"
                            >
                              <span
                                color="primary"
                              >
                                Forgot your Username or Password 
                              </span>
                            </span>
                            <span
                              class="MuiTouchRipple-root"
                            />
                          </button>
                          <button
                            class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary Mui-disabled Mui-disabled"
                            disabled=""
                            tabindex="-1"
                            type="button"
                          >
                            <span
                              class="MuiButton-label"
                            >
                              Signin
                            </span>
                          </button>
                        </div>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
              <div
                class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-12 MuiGrid-grid-md-5 MuiGrid-grid-lg-5 MuiGrid-grid-xl-5"
              >
                <div
                  class="MuiBox-root MuiBox-root-402 makeStyles-backgroundImg-217"
                />
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        class="MuiGrid-root makeStyles-footerStyle-6 MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-12 MuiGrid-grid-md-12 MuiGrid-grid-lg-12 MuiGrid-grid-xl-12"
      />
    </div>
  </div>
   
</DocumentFragment>
`;
